pages/auth/join.tsx

```diff
@@ -31,6 +35,31 @@ const Signup: NextPageWithLayout<inferSSRProps<typeof getServerSideProps>> = ({
           <Join />
         )}
       </div>
+      <div className="divider">or sign up with</div>
+      <Link href="/api/auth/signin">
```

Comment:
Do we need both Link href and onClick action?

Reply:
`onClick` action is required for the operation, `href` prop in mandatory for the `Link` component.
A NextAuth.js example using both `href` and `onclick` event: [Example](https://github.com/nextauthjs/next-auth-example/blob/main/components/header.tsx)

Reply:
Can we use button instead?

Reply:
Resolved, updated links to buttons.